src/test/java/step_defs/US2Steps.java

```diff
@@ -79,9 +81,18 @@ public void userShouldSeeAnInputFieldInitialDepositAmount() {
 
 
     @Then("^user should see \"([^\"]*)\" button$")
-    public void user_should_see_button(String submit) {
-        Assert.assertTrue("Submit button is not displayed", newCheckingAccountPage.submitButton.isDisplayed());
-        Assert.assertTrue("Submit button doesn't show SUBMIT",
-                newCheckingAccountPage.submitButton.getText().contains(submit));
+    public void user_should_see_button(String button) {
```

Comment:
it supposed to be CamelCase, not undescore
```suggestion
    public void userShouldSeeButton(String button) {
```